addons/project/models/project.py

```diff
@@ -60,6 +62,11 @@ def unlink(self):
                 stages = self.filtered(lambda x: x not in shared_stages)
         return super(ProjectTaskType, stages).unlink()
 
+    @api.depends('project_ids', 'project_ids.rating_active')
+    def _compute_project_rating_disabled_warning(self):
+        for r in self:
```

Comment:
``for stage in self`` It's always better when you clearly see on what kind of record you work.

Reply:
Done